package.json

```diff
@@ -25,7 +30,7 @@
   "scripts": {
     "build-clean": "rimraf ./packages/*/build ./packages/*/build-module",
     "build": "node ./scripts/build.js",
-    "postinstall": "lerna bootstrap && npm run build",
+    "postinstall": "lerna bootstrap && check-node-version --package && npm run build",
```

Comment:
Rather than include `check-node-version --package` in `postinstall` can you add it as `prebuild`, that will ensure it is run prior to `build`, `publish:dev`, and `publish:prod` scripts.

(For those who don't use nvm or for myself where nvm here on my system can be troublesome at times (I use Fish shell rather than Bash) and it doesn't always switch Node.js version automattically)

Reply:
Works for me :)

Reply:
Actually, I will add additional checks to be on the safe side.

Reply:
Addressed with https://github.com/WordPress/packages/pull/61/commits/daf94907760563ab32fa6b01b686e47f928a0ba6.

Adding it as `prebuild` works for all cases, as all of them execute `npm run build`. Great idea 🙇